qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/ParameterizedRollingUpgradeTestCase.java

```diff
@@ -96,6 +98,8 @@ public void upgradeNode() throws Exception {
         // Skip remaining tests if upgrade failed
         assumeFalse("Cluster upgrade failed", upgradeFailed);
 
+        beforeUpgrade();
```

Comment:
I checked a few test failures and it fails with fails with indices/data streams that are managed internally. For example the ilm history or deprecation logging. 

I think without this change a node upgrade could happen before the `@Before` method with old feature check was invoked. In the case an internal index is created while the upgrade is in progress. Older nodes don't understand the renamed index setting. This caused the test to fail before it got the old feature check. (cluster would never go into a green state after a node upgraded)

So I think this should ensure that this doesn't happen.